packages/ui/primitives/signature-pad/signature-pad.tsx

```diff
@@ -33,7 +35,8 @@ export const SignaturePad = ({
   const $el = useRef<HTMLCanvasElement>(null);
 
   const [isPressed, setIsPressed] = useState(false);
-  const [points, setPoints] = useState<Point[]>([]);
+  const [lines, setLines] = useState<Point[][]>([]);
```

Comment:
Any reason to rename to lines?

Reply:
I thought that it would be better in understanding speacially for beginners

Reply:
This should be kept as Point as the type is `Point`